src/encoded/types/analyte.py

```diff
@@ -1,7 +1,27 @@
 from snovault import collection, load_schema
+from snovault.util import debug_log
 
-from .submitted_item import SubmittedItem
+from pyramid.view import view_config
 
+from .submitted_item import (
+    SUBMITTED_ITEM_ADD_VALIDATORS,
+    SUBMITTED_ITEM_EDIT_PATCH_VALIDATORS,
+    SUBMITTED_ITEM_EDIT_PUT_VALIDATORS,
+    SubmittedItem,
+)
+
+from .base import (
+    collection_add,
+    item_edit,
+    Item
+)
+
+
+RNA_PROPERTIES = [
+    'rna_integrity_number',
+    'rna_integrity_number_instrument',
+    'ribosomal_rna_ratio'
+]
```

Comment:
While it may be fragile - perhaps this and Will's comment about wanting to check dna props may be adressable if the convention of specifically including dna or rna is those props that are specific for one or the other is maintained/enforced and the validator made more general so both cases can be handled.

Should be weighed against the possible fragility and maintainability but seems a reasonable idea on initial glance at schema and intent?

Reply:
I like this idea. I implemented in the new commit and I don't think it will be terribly difficult to maintain